tools/clang/lib/Parse/ParseDecl.cpp

```diff
@@ -616,6 +616,10 @@ bool Parser::MaybeParseHLSLAttributes(std::vector<hlsl::UnusualAnnotation *> &ta
       ConsumeToken(); // consume colon.
 
       StringRef semanticName = Tok.getIdentifierInfo()->getName();
+      if (semanticName.equals("VFACE")) {
```

Comment:
If (and this isn't sure) you want to keep the Gec automatic conversion, this should only fire if `getLangOpts().EnableDX9CompatMode` is false. Since otherwise the conversion will occur and produce its own warning.

Reply:
I just removed the change for `Gec`